validation/WorkZoneFeed.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
from here: https://github.com/usdot-jpo-ode/wzdx/blob/main/schemas/4.2/WorkZoneFeed.json

Maybe I should just reference the repo rather than keep a local copy?